src/Site_Command.php

```diff
@@ -521,11 +806,12 @@ private function delete_site() {
 		}
 
 		if ( is_dir( $this->site_root ) ) {
-			if ( ! \EE\Utils\default_launch( "sudo rm -rf $this->site_root" ) ) {
+			if ( ! \EE\Utils\default_launch( "rm -rf $this->site_root" ) ) {
```

Comment:
I think before doing this we should first check if `$this->docker::docker_compose_down( $this->site_root )` command ran successfully. ( Or is it exit with error from this function itself if its failed ?)

Reply:
Yes. Good idea. But instead of sending when it fails, we've opted for a better approach.
While creating site we'll tag the containers with two labels
created_by=EasyEngine
site_name=<sitename>

Then if `docker-compose down` fails, we'll search for all containers with above two tags and stop them.

Hence we'll be able to continue deletion